introduction.md

```diff
@@ -0,0 +1 @@
+<p>Hi I am V. and pretty friggin awesome and so are you </p>
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.